src/yardstick/utils/__init__.py

```diff
@@ -51,7 +51,7 @@ def try_convert_int(s: str) -> int | None:
     components = vuln_id.split("-")
 
     if components and len(components) >= 2:
-        if components[0].lower() in {"cve", "alas"}:
+        if components[0].lower() in {"cve", "alas", "elsa"}:
```

Comment:
@westonsteimel wanted to raise up this explicit change -- is this correct? (also check out the test changes as well)

Reply:
yeah, I think so